pkg/jq/jq.go

```diff
@@ -65,6 +75,10 @@ func EvaluateFormatted(input io.Reader, output io.Writer, expr string, indent st
 			break
 		}
 		if err, isErr := v.(error); isErr {
+			var e *gojq.HaltError
+			if errors.As(err, &e) && e.Value() == nil {
```

Comment:
For anyone else wondering what this is about, from the `gojq` docs:

> In any case, it is recommended to stop the iteration on [gojq.HaltError](https://pkg.go.dev/github.com/itchyny/gojq#HaltError), which is emitted by halt and halt_error functions, although these functions are rarely used. The error implements [gojq.ValueError](https://pkg.go.dev/github.com/itchyny/gojq#ValueError), and if the error value is nil, stop the iteration without handling the error. Technically speaking, we can fix the iterator to terminate on the halting error, but it does not terminate at the moment. The halt function in jq not only stops the iteration, but also terminates the command execution, even if there are still input values. So, gojq leaves it up to the library user how to handle the halting error.

This is handing the case when the `jq` expression contains [`halt`](https://jqlang.github.io/jq/manual/v1.6/#halt) (exit with no error), which previously we wouldn't have respected.